new_dawn_server/users/api/tests/test_users_api.py

```diff
@@ -181,17 +181,18 @@ def test_user_profile_get_with_height_preference(self):
                                        "height": 182,
                                    })
 
-        res = self.api_client.get("/api/v1/profile/?height__range=175,180", format="json")
+        res = self.api_client.get("/api/v1/profile/?ranking=True&viewer_id=2", format="json")
```

Comment:
Can you add a separate test function to test recommendation? We should decouple tastiepie default filtering and your customized recommendation.